### PR TITLE
upcoming v7.8.0

### PR DESCRIPTION
**Pull request by matizm**

### CODE REVIEW

File: rst/changelog/changelog.rst [hunk 1]
```diff
@@ -6,9 +6,56 @@ Changelog
...
+- [PC] LogPlayerKill.dBNOId
+- [PC] LogPlayerKill.victimGameResult
+- [PC] LogPlayerMakeGroggy.damageCauserAdditionalInfo
+- [PC] LogPlayerMakeGroggy.damageReason
+- [PC] LogPlayerRevive.dBNOId
+- [PC] LogVehicleLeave.maxSpeed
+
+Data Changes:
+
+- [PC] The leaderboards endpoint will be capable of responding with the top 1000 players for each game mode in increments of 100.
```
Comment (jaloisi834) [inline rst/changelog/changelog.rst]: I think people might get confused by this. Maybe add that we will share specifics soon?

Comment (matizm) [reply rst/changelog/changelog.rst]: I left it out of the changelog, but I'm including details in the announcement and emails and forum post

File: swagger/en/schemas/gameModeStats.yml [hunk 2]
```diff
@@ -74,6 +74,10 @@ properties:
     type: number
     description: "[PC only] Number of rank points the player was awarded. This value will be 0 when roundsPlayed < 10"
     minimum: 0
+  rankPointsTitle:
+    type: string
+    description: "[PC Only, Available Dec 19, 2018] Rank title in the form title-level"
+    minimum: "0-0"
```
Comment (jaloisi834) [inline swagger/en/schemas/gameModeStats.yml]: I don't think this is the intended use of the minimum attribute. Does this even work? I'm pretty sure it has to be a number.

File: rst/changelog/changelog.rst (hunk 1 above)
Comment (richardardrichard) [reply rst/changelog/changelog.rst]: We probably will just make this transparent to users. Let's not announce this yet.

**Pull request merged**